### ISSUE TITLE
Software for large-scale survey projects

### ISSUE THREAD

**Issue by lucaarcteam** [archeos/ArcheOS#138]
We need a software for large-scale survey projects. We can consider:
1. Polygontool (developed for us by Szabo Koello): https://github.com/keulemaster/Polygontool/issues
2. ArcheoCAD (by INRAP, the French archaeological statal institution);
3. a tool related with gvSIG, developed by Benjamin Ducke (actually I do not remember the name)

Up to now we just tested Polygontool (it worked very good) for large-scale survey. I tested ArcheoCAD just with small file, but it seems could replace Polygontool. ArcheoCAd should in case be placed into the archeos-qgis-plugin package. We need feedback for the third option

**Comment by fabfurnari** [comment]
Why not both ArcheoCAD and Polygontool?

**Comment by lucaarcteam** [comment]
yes, would be probably the bset option. I think it is note (yet?) possible to do really everything just with ArcheoCAD
